README.md

```diff
@@ -19,7 +19,7 @@ To analyze a log file, run the script with the following command line arguments:
 
 Example usage:
 
-    logdetective https://example.com/logs.txt --model https://mymodel.co/model_file --summarizer drain --nlines 10
```

Comment:
That's actually correct.

Reply:
Really? It didn't worked for me and based on the source code it should be as `--n_lines`:

https://github.com/fedora-copr/logdetective/blob/master/logdetective/logdetective.py#L176

I'm getting:

```
logdetective: error: unrecognized arguments: --nlines 10
```

Reply:
Sorry I misread the diff.